flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/DashboardConfigurationHeaders.java

```diff
@@ -28,7 +28,7 @@
  */
 public final class DashboardConfigurationHeaders implements MessageHeaders<EmptyRequestBody, DashboardConfiguration, EmptyMessageParameters> {
 
-	private static final DashboardConfigurationHeaders INSTANCE = new DashboardConfigurationHeaders();
+	public static final DashboardConfigurationHeaders INSTANCE = new DashboardConfigurationHeaders();
```

Comment:
is this class used anywhere? Because before your change it looked like there is no way to use it at all

Reply:
This is the exact code from the master flink branch PR. I havent made any changes on top of that.